typings/sharify.d.ts

```diff
@@ -22,6 +22,7 @@ declare module "sharify" {
       readonly METAPHYSICS_ENDPOINT: string
       readonly NODE_ENV: string
       readonly XAPP_TOKEN: string
+      readonly ENABLE_NEW_BUY_NOW_FLOW: string
```

Comment:
This sort of should be a boolean, but since these all start out as env vars (which are always inherently strings themselves, when set in the environment), I figured typing it as a string (and we're only using it in a truthy check) makes the most sense.